scripts/build_native_deps.sh

```diff
@@ -440,10 +408,33 @@ build_python()
         "$src/configure" --prefix="$PYTHON_INSTALL" --disable-shared --enable-static
         $MAKE -j$PROCS
         $MAKE install
-    ) > "$LOG/python.log" 2>&1
+    ) > "$LOG/python.log" 2>&1 || print_failed_and_exit "$LOG/python.log"
+)
+
+BUILD_FUNCTIONS=(
+    build_expat
+    build_freetype2
+    build_util_linux
+    build_fontconfig
+    build_ghostscript
+    build_libffi
+    build_zlib
+    build_glib2
+    build_gmp
+    build_libtool
+    build_guile
+    build_pixman
+    build_cairo
+    build_harfbuzz
+    build_pango
+    build_python
 )
-build_python
-echo ""
+
+for fn in ${BUILD_FUNCTIONS[@]}; do
+    echo ""
+    $fn
+done
```

Comment:
What is the advantage of doing this in a loop?

Reply:
None. I probably wrote it like that out of habit.

Reply:
Fixed in commit f0a658b